### PR TITLE
SX2-1699 Add animal bulk to slate docu.

### PR DESCRIPTION
**Pull request by smaXtecStefan**
added animal bulk upload

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -408,6 +408,102 @@ This endpoint retrieves from a provided `organisation_id` all animals.
...
+
+> Request
+
+```python
+import requests
+
+endpoint = 'https://api-staging.smaxtec.com/integration/v2'
+organisation_id = '123456qwertz'
+optional_parameters = '?include_archived=false'
+route = endpoint + '/organisations/' + organisation_id + '/animals' + optional_parameters
```
Comment (smaXtecStefan) [inline source/index.html.md]: the route needs to be without the `optional_parameters`.

File: source/index.html.md [hunk 2]
```diff
@@ -408,6 +408,102 @@ This endpoint retrieves from a provided `organisation_id` all animals.
...
+token = 'yx2zvuB8JD8ppwGti84OT8Muq5eiB2b2EZqsqC-HOXUvLSg'
+user_agent = 'HerdManagerSoftwareXYZ/2.4'
+headers = {
+    'accept': 'application/json',
+    'User-Agent': user_agent,
+    'Authorization': 'bearer ' + token
+}
+
+r = requests.put(route, headers=headers)
```
Comment (smaXtecStefan) [inline source/index.html.md]: the put call needs the `json=data` parameter where data are the animals which should get integrated into our system

File: source/index.html.md [hunk 3]
```diff
@@ -408,6 +408,102 @@ This endpoint retrieves from a provided `organisation_id` all animals.
...
+
+status_code = r.status_code
+all_animals = r.json()
+```
+
+```bash
+curl -X GET "[endpoint]/organisations/[organisation_id]/animals" \
+        -H  "accept: application/json" \
+        -H  "Authorization: bearer [token]" \
+        -H  "User-Agent: [user_agent]"
```
Comment (smaXtecStefan) [inline source/index.html.md:454]: the `-d ...` flag needs to be added as well

File: source/index.html.md [hunk 4]
```diff
@@ -408,6 +408,102 @@ This endpoint retrieves from a provided `organisation_id` all animals.
...
+**HTTP Request**
+
+`PUT "[endpoint]/organisations/[organisation_id]/animals"`
+
+**URL Parameters**
+
+| Parameter        | Description                                         |                  |
+| ---------------- | --------------------------------------------------- | ---------------- |
+| organisation_id  | ID of the organisation where the animal belongs to. | `required`       |
+| include_archived | Include archived animals.                           | default: `false` |
```
Comment (smaXtecStefan) [inline source/index.html.md]: the `include_archived` isn't an accepted parameter. this needs to be deleted.

instead the animals should be put here. you can reference to the parameters from **Create/Update Single Animal**

File: source/index.html.md [hunk 5]
```diff
@@ -408,6 +408,102 @@ This endpoint retrieves from a provided `organisation_id` all animals.
 | organisation_id  | ID of the organisation where the animal belongs to. | `required`       |
 | include_archived | Include archived animals.                           | default: `false` |
 
+## Put Animals
```
Comment (smaXtecStefan) [inline source/index.html.md]: to be consistent with the other subsections please change this to **Create/Update Multiple Animals**

File: source/index.html.md (hunk 3 above)
Comment (KrennKristof) [reply source/index.html.md:454]: in line 455 there is one " missing at organisation_id

**Review by KrennKristof — CHANGES_REQUESTED** [review]

**Comment by KrennKristof** [comment]
closed, documentation moved to gitlab

**Pull request closed without merging**